pkg/kubelet/apis/podresources/server_v1.go

```diff
@@ -23,8 +23,10 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	kubefeatures "k8s.io/kubernetes/pkg/features"
 	"k8s.io/kubernetes/pkg/kubelet/metrics"
+	"k8s.io/kubernetes/pkg/kubelet/types"
 
-	"k8s.io/kubelet/pkg/apis/podresources/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/kubelet/pkg/apis/podresources/v1"
```

Comment:
not-so-nit: can we rename "podresourcesv1"? "v1" is usually (= almost always) an alias for "api/core/v1" so I'd keep the principle of least surprise

Reply:
done